obspy/signal/spectral_estimation.py

```diff
@@ -1317,7 +1324,13 @@ def _load(data):
                 # we have to convert those back to lists (or simple types), so
                 # that additionally processed data can be appended/inserted
                 # later.
-                data_ = data[key]
+                try:
+                    data_ = data[key]
+                except ValueError:
+                    msg = ("Loading PPSD results saved with obspy versions < "
```

Comment:
```suggestion
                    msg = ("Loading PPSD results saved with ObsPy versions < "
```

Reply:
We could bump the internal version number and use that to determine problems with unpickling.